demo/site/src/recursivelyLoadBlockData.ts

```diff
@@ -0,0 +1,9 @@
+import { recursivelyLoadBlockData as cometRecursivelyLoadBlockData } from "@comet/cms-site";
+import { GraphQLClient } from "graphql-request";
+
+import blocksMeta from "../block-meta.json";
```

Comment:
This includes the complete block meta bundle in the client-side bundle (because of it's usage in `demo/site/src/pages/preview/admin/page.tsx`. Could this be a problem?

Reply:
True, I haven't thought about that.

I could lazy load it?

Reply:
> I could lazy load it?

I just tested it locally in one of our projects. The bundle size shared across all pages is increased about 11kb. I believe it's a good idea to lazy load it.